### DISCUSSION TITLE
Dropdown items color and size

### DISCUSSION THREAD

**Discussion by francescotau** [saadeghi/daisyui#1852] [Q&A] [answered]
Hi, how to change dropdown items style? For example set danger color and sm size. Thank you

**Comment by saadeghi** [comment, marked as answer]
`dropdown` class itself has no design. It just opens the content when clicked:
```
<div class="dropdown">
  <label tabindex="0">Click</label>
  <div tabindex="0" class="dropdown-content">
    content
  </div>
</div>
```
On the examples, you can see I used `btn` and `menu` and some utility classes but you can use anything else:
```
<div class="dropdown">
  <label tabindex="0" class="btn m-1">Click</label>
  <ul tabindex="0" class="dropdown-content menu p-2 shadow bg-base-100 rounded-box w-52">
    <li><a>Item 1</a></li>
    <li><a>Item 2</a></li>
  </ul>
</div>
```

For `danger` color you can use `text-danger` (or `bg-danger text-danger-content` for background)

For small text size: `text-sm`. Or if you meant small button, `btn-sm`
